docs/manage/manage/data-regions.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Choose data region"
+linkTitle: "Choose data region"
+description: "Configure where in the world Viam stores your cloud data."
+weight: 30
+type: "docs"
+tags: ["data region", "region", "data continent", "compliance", "performance"]
+---
+
+You can specify the geographical location where Viam stores your data.
+
+## Introduction
```

Comment:
We don't do this anywhere else and it duplicates the above. so can just merge the two
```suggestion
```